lib/expeditor/command.rb

```diff
@@ -115,13 +115,13 @@ def reset_fallback(&block)
       @fallback_var = Concurrent::IVar.new
       @normal_future.add_observer do |_, value, reason|
         if reason != nil
-          future = RichFuture.new(executor: Concurrent.configuration.global_task_pool) do
+          future = RichFuture.new(executor: Concurrent.global_io_executor) do
             success, val, reason = Concurrent::SafeTaskExecutor.new(block, rescue_exception: true).execute(reason)
-            @fallback_var.complete(success, val, reason)
+            @fallback_var.send(:complete, success, val, reason)
```

Comment:
Read the commit message https://github.com/k0kubun/expeditor/commit/526cbdc7de6960f7f20b8f902c0d94d14d1bcabe